core/templates/dev/head/css/oppia.css

```diff
@@ -3921,17 +3921,26 @@ md-card.preview-conversation-skin-supplemental-card {
   font-size: 1em;
   font-weight: normal;
   line-height: 1.2em;
-  margin: 0.3em 0.5em;
+  margin: 0.3em 0.5em;  
   padding: 0.2em;
   position: absolute;
   word-break: break-all;
 }
-
 .oppia-activity-summary-tile .objective {
-  height: 60px;
+  display: -webkit-box;
   font-family: "Rubik", "Roboto", Arial, sans-serif;
   font-size: 0.85em;
+  height: 70px;
+  line-height: 1.26;
+  /* `max-height` is calculated as: 1.26 (line-height) * 3 (lines) / 0.85
+  (font-size). An additional (1.26 (line-height) * 0.85 (font-size) * 0.25) is added
+  to account for letters (like y,j and g) that fall beneath the baseline. */
```

Comment:
Ah, OK. So is the behaviour different for webkit and non-webkit browsers?

That is, let's say we get rid of -webkit-line-clamp, but left max-height in place. In webkit browsers, what would change? In non-webkit browsers, what would change?